### PR TITLE
Cleanup tour

### PR DESCRIPTION
**Pull request by rht**
#555

### CODE REVIEW

File: core/commands/tour.go [hunk 1]
```diff
@@ -64,11 +62,13 @@ func tourRunFunc(req cmds.Request, res cmds.Response) {
 		fmt.Fprintln(&w, err)
 		fmt.Fprintln(&w, "")
 		fprintTourList(&w, tour.TopicID(cfg.Tour.Last))
+		res.SetOutput(bytes.NewReader(w.Bytes()))
 
 		return
 	}
 
 	fprintTourShow(&w, t)
+	res.SetOutput(bytes.NewReader(w.Bytes()))
```
Comment (rht) [inline core/commands/tour.go]: Is there a better way to link the ```io.Reader``` and ```io.Writer``` together without having to create a buffer? I tried ```io.Pipe``` but it didn't work.

Comment (whyrusleeping) [inline core/commands/tour.go]: you should just be able to do `res.SetOutput(w)` here

File: core/commands/tour.go [hunk 2]
```diff
@@ -50,7 +49,6 @@ func tourRunFunc(req cmds.Request, res cmds.Response) {
 	}
 
 	var w bytes.Buffer
```
Comment (whyrusleeping) [inline core/commands/tour.go]: it would make me really happy if we could change this to `w := new(bytes.Buffer)`

Comment (rht) [inline core/commands/tour.go]: ok let me change these all across the repo.

**Comment by whyrusleeping** [comment]
:+1: woo!

**Comment by jbenet** [comment]
wonder if tour should be a standalone tool altogether, instead of tied in...

**Comment by wking** [comment]
On Thu, May 28, 2015 at 11:24:03PM -0700, Juan Batiz-Benet wrote:
> wonder if tour should be a standalone tool altogether, instead of
> tied in...

Eventually, I'd like to see the tour and other docs just bundled into
an archive ([1], #1195).  Unpacking the docs to your local file system
and reading them with your pager/editor/browser seems like a better UI
than trying to write a file-browser command.  But I don't have a
problem with folks polishing the existing tour command until we get
around to writing the archive tooling.

[1]: https://github.com/ipfs/archive-format/issues

**Pull request merged**